opta/commands/deploy.py

```diff
@@ -81,6 +78,8 @@ def deploy(
     pre_check()
 
     config = check_opta_file_exists(config)
+    if local:
+        config = _local_setup(config, image_tag=tag, refresh_local_env=True)
```

Comment:
now that local is supported with all commands, would that make sense for `check_opta_file_exists(config)` to manage the local config and remove the logic from the caller?

```
    config = check_opta_file_exists(config)
    if local:
        config = _local_setup(config, image_tag=tag, refresh_local_env=True)
```
would become
```
    # local could be an optional parameter 
    config, env = check_opta_file_exists(config, local)
```